rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -112,12 +116,123 @@ void setUp() {
         );
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("Validation failure tests")
+    class ValidationFailureTests {
+        @BeforeEach
+        void setUp() {
+            logs = new ArrayList<>();
+            BridgeEventLogger bridgeEventLogger = new BridgeEventLoggerImpl(
+                bridgeMainNetConstants,
+                allActivations,
+                logs
+            );
+
+            InMemoryStorage storageAccessor = new InMemoryStorage();
+            FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+            federationStorageProvider.setProposedFederation(proposedFederation);
+
+            federationSupport = new FederationSupportImpl(
+                bridgeMainNetConstants.getFederationConstants(),
+                federationStorageProvider,
+                rskExecutionBlock,
+                allActivations
+            );
+
+            bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeMainNetConstants)
+                .withProvider(bridgeStorageProvider)
+                .withEventLogger(bridgeEventLogger)
+                .withActivations(allActivations)
+                .withFederationSupport(federationSupport)
+                .withFeePerKbSupport(feePerKbSupport)
+                .withExecutionBlock(rskExecutionBlock)
+                .build();
+        }
+
+        @Test
+        void processValidationFailure_whenSvpFundTxHashUnsigned_shouldLogValidationFailureAndClearValue() {
+            // arrange
+            Sha256Hash svpFundTxHashUnsigned = BitcoinTestUtils.createHash(1);
+            bridgeStorageProvider.setSvpFundTxHashUnsigned(svpFundTxHashUnsigned);
+
+            // act
+            bridgeSupport.processValidationFailure(proposedFederation);
+
+            // assert
+            assertLogCommitFederationFailed();
+            assertProposedFederationWasCleared();
+            assertFalse(bridgeStorageProvider.getSvpFundTxHashUnsigned().isPresent());
+        }
+
+        @Test
+        void processValidationFailure_whenSvpFundTxSigned_shouldLogValidationFailureAndClearValue() {
+            // arrange
+            BtcTransaction svpFundTx = mock(BtcTransaction.class);
```

Comment:
```suggestion
            BtcTransaction svpFundTx = new BtcTransaction();
```
Just to avoid the usage of mocks

Reply:
Same in other tests

Reply:
updated